src/ResponseTagger/Delegator/XLocationIdBCTagger.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license   For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\PlatformHttpCacheBundle\ResponseTagger\Delegator;
+
+use EzSystems\PlatformHttpCacheBundle\ResponseConfigurator\ResponseCacheConfigurator;
+use EzSystems\PlatformHttpCacheBundle\ResponseTagger\ResponseTagger;
+use Symfony\Component\HttpFoundation\Response;
+
+class XLocationIdBCTagger implements ResponseTagger
+{
+    /**
+     * @var ResponseTagger
+     */
+    private $locationTagger;
+
+    const LOCATION_ID_HEADER = 'X-Location-Id';
+
+    /**
+     * XLocationIdTagger constructor.
+     *
+     * @param ResponseTagger $locationTagger
+     */
+    public function __construct(ResponseTagger $locationTagger) {
+        $this->locationTagger    = $locationTagger;
+    }
+
+    public function tag(ResponseCacheConfigurator $configurator, Response $response, $view)
+    {
+        if ($view !== null || !$response->headers->has(static::LOCATION_ID_HEADER)) {
+            return null;
+        }
+
+        $locationIds = explode("|", $response->headers->get(static::LOCATION_ID_HEADER));
+        foreach ($locationIds as $locationId) {
+            $location = $this->repository->getLocationService()->loadLocation($locationId);
+            $this->locationTagger->tag($configurator, $response, $location);
+            $response->headers->remove('X-Location-Id');
```

Comment:
You should replace `'X-Location-ID'` with `self::LOCATION_ID_HEADER`.

Reply:
good catch! Fixed.